src/System.Windows.Forms/src/System/Windows/Forms/DataGridView/DataGridViewCell.cs

```diff
@@ -2571,64 +2542,55 @@ public virtual void InitializeEditingControl(int rowIndex, object initialFormatt
-            dgv.EditingControlAccessibleObject.SetParent(AccessibilityObject);
+            dgv.EditingControlAccessibleObject!.SetParent(AccessibilityObject);
             AccessibilityObject.SetDetachableChild(dgv.EditingControlAccessibleObject);
         }
     }
 
-    protected virtual bool KeyDownUnsharesRow(KeyEventArgs e, int rowIndex)
-    {
-        return false;
-    }
+    protected virtual bool KeyDownUnsharesRow(KeyEventArgs e, int rowIndex) => false;
 
-    internal bool KeyDownUnsharesRowInternal(KeyEventArgs e, int rowIndex)
-    {
-        return KeyDownUnsharesRow(e, rowIndex);
-    }
+    internal bool KeyDownUnsharesRowInternal(KeyEventArgs e, int rowIndex) =>
+        KeyDownUnsharesRow(e, rowIndex);
 
-    public virtual bool KeyEntersEditMode(KeyEventArgs e)
-    {
-        return false;
-    }
+    public virtual bool KeyEntersEditMode(KeyEventArgs e) => false;
 
-    protected virtual bool KeyPressUnsharesRow(KeyPressEventArgs e, int rowIndex)
-    {
-        return false;
-    }
+    protected virtual bool KeyPressUnsharesRow(KeyPressEventArgs e, int rowIndex) => false;
 
-    internal bool KeyPressUnsharesRowInternal(KeyPressEventArgs e, int rowIndex)
-    {
-        return KeyPressUnsharesRow(e, rowIndex);
-    }
+    internal bool KeyPressUnsharesRowInternal(KeyPressEventArgs e, int rowIndex) =>
+        KeyPressUnsharesRow(e, rowIndex);
 
-    protected virtual bool KeyUpUnsharesRow(KeyEventArgs e, int rowIndex)
-    {
-        return false;
-    }
+    protected virtual bool KeyUpUnsharesRow(KeyEventArgs e, int rowIndex) => false;
 
-    internal bool KeyUpUnsharesRowInternal(KeyEventArgs e, int rowIndex)
-    {
-        return KeyUpUnsharesRow(e, rowIndex);
-    }
+    internal bool KeyUpUnsharesRowInternal(KeyEventArgs e, int rowIndex) =>
+        KeyUpUnsharesRow(e, rowIndex);
 
-    protected virtual bool LeaveUnsharesRow(int rowIndex, bool throughMouseClick)
-    {
-        return false;
-    }
+    protected virtual bool LeaveUnsharesRow(int rowIndex, bool throughMouseClick) => false;
 
-    internal bool LeaveUnsharesRowInternal(int rowIndex, bool throughMouseClick)
-    {
-        return LeaveUnsharesRow(rowIndex, throughMouseClick);
-    }
+    internal bool LeaveUnsharesRowInternal(int rowIndex, bool throughMouseClick) =>
+        LeaveUnsharesRow(rowIndex, throughMouseClick);
 
     [EditorBrowsable(EditorBrowsableState.Advanced)]
-    public static int MeasureTextHeight(Graphics graphics, string text, Font font, int maxWidth, TextFormatFlags flags)
-    {
-        return DataGridViewCell.MeasureTextHeight(graphics, text, font, maxWidth, flags, out bool widthTruncated);
-    }
+    public static int MeasureTextHeight(
+        Graphics graphics,
+        string? text,
+        Font font,
+        int maxWidth,
+        TextFormatFlags flags) => MeasureTextHeight(
+            graphics,
+            text,
+            font,
+            maxWidth,
+            flags,
+            out bool widthTruncated);
```

Comment:
It looks like this is unused
```suggestion
            out _);
```